### ISSUE TITLE
bravado on pypi does not offer extra 'fido'

### ISSUE THREAD

**Issue by ivanlei** [Yelp/bravado#277]
```shell
(default) ➜  ~ pip install 'bravado[fido]'
Collecting bravado[fido]
  Downloading bravado-8.4.0.tar.gz
  bravado 8.4.0 does not provide the extra 'fido'
  ...
```

**Comment by macisamuele** [comment]
@ivanlei The feature you're looking for was introduced by PR #273 and it is not yet released.
The latest bravado version is 8.4.0 released on September 2016.
Soon we'll release the new version that will define ``fido`` as optional (extra) dependency.

**Comment by ngaya-ll** [comment]
@macisamuele Is there a target release date for the new version?

**Comment by sjaensch** [comment]
I just released version 9.0.0.
